config.json

```diff
@@ -352,8 +352,8 @@
 
         "shield": {
             "energy_source": "energy",
-            "maintenance_factor": 0.1,
-            "regeneration_factor": 0.1
+            "maintenance_factor": 0.05,
+            "regeneration_factor": 0.05
```

Comment:
My starting Llama’s shields weren’t recharging. They were taking too much power to maintain, let alone replenish. These two settings changes fix that.